<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="bgwidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>800</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget#bgwidget{background-color:qlineargradient(spread:pad, x1:0.149, y1:0.130364, x2:0.93, y2:0.942727, stop:0 rgba(218, 100, 255, 1), stop:0.9801 rgba(255, 255, 255, 255));}</string>
   </property>
   <widget class="QLabel" name="Label_A">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>30</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Consolas</family>
      <pointsize>22</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>KEY:</string>
    </property>
   </widget>
   <widget class="QLabel" name="Label_B">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>80</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Consolas</family>
      <pointsize>19</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Jazyk</string>
    </property>
   </widget>
   <widget class="QLabel" name="Label_Input">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>210</y>
      <width>151</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Consolas</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Vstupní text:</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="CheckBox_Sifrovat">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>130</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Consolas</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Šifrovat</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">Execute</string>
    </attribute>
   </widget>
   <widget class="QRadioButton" name="CheckBox_Desifrovat">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>130</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Consolas</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Dešifrovat</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">Execute</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="Button_Execute">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>290</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Consolas</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="plainTextEdit_A">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>30</y>
      <width>261</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Consolas</family>
     </font>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="plainTextEdit_Input">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>200</y>
      <width>451</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Shell Dlg 2</family>
      <pointsize>8</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 8pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelVysledek">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>420</y>
      <width>1181</width>
      <height>331</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Shell Dlg 2</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
    <property name="margin">
     <number>0</number>
    </property>
    <property name="textInteractionFlags">
     <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
    </property>
   </widget>
   <widget class="QRadioButton" name="CheckBox_JazykCZ">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>90</y>
      <width>95</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Consolas</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>CZ</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
    <property name="autoExclusive">
     <bool>false</bool>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">Jazyk</string>
    </attribute>
   </widget>
   <widget class="QRadioButton" name="CheckBox_JazykEN">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>90</y>
      <width>95</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Consolas</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>EN</string>
    </property>
    <property name="autoExclusive">
     <bool>false</bool>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">Jazyk</string>
    </attribute>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>40</y>
      <width>261</width>
      <height>161</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777206</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Consolas</family>
     </font>
    </property>
    <property name="dragEnabled">
     <bool>false</bool>
    </property>
    <property name="dragDropOverwriteMode">
     <bool>false</bool>
    </property>
    <property name="dragDropMode">
     <enum>QAbstractItemView::NoDragDrop</enum>
    </property>
    <property name="defaultDropAction">
     <enum>Qt::MoveAction</enum>
    </property>
    <property name="showGrid">
     <bool>true</bool>
    </property>
    <property name="sortingEnabled">
     <bool>true</bool>
    </property>
    <attribute name="horizontalHeaderVisible">
     <bool>false</bool>
    </attribute>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>50</number>
    </attribute>
    <attribute name="verticalHeaderVisible">
     <bool>false</bool>
    </attribute>
    <row>
     <property name="text">
      <string/>
     </property>
    </row>
    <row>
     <property name="text">
      <string/>
     </property>
    </row>
    <row>
     <property name="text">
      <string/>
     </property>
    </row>
    <row>
     <property name="text">
      <string/>
     </property>
    </row>
    <row>
     <property name="text">
      <string/>
     </property>
    </row>
    <column>
     <property name="text">
      <string/>
     </property>
    </column>
    <column>
     <property name="text">
      <string/>
     </property>
    </column>
    <column>
     <property name="text">
      <string/>
     </property>
    </column>
    <column>
     <property name="text">
      <string/>
     </property>
    </column>
    <column>
     <property name="text">
      <string/>
     </property>
    </column>
    <item row="0" column="0">
     <property name="text">
      <string/>
     </property>
    </item>
    <item row="0" column="1">
     <property name="text">
      <string/>
     </property>
    </item>
    <item row="0" column="2">
     <property name="text">
      <string/>
     </property>
    </item>
    <item row="0" column="3">
     <property name="text">
      <string/>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>130</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Consolas</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Funkce</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>160</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Consolas</family>
      <pointsize>19</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Typ</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="matica_5X5">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>170</y>
      <width>95</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Consolas</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>ADFGX</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="matica_6X6">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>170</y>
      <width>95</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Consolas</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>ADFGVX</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_Generate">
    <property name="geometry">
     <rect>
      <x>1010</x>
      <y>60</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Consolas</family>
      <pointsize>13</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Generovat</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_Reset">
    <property name="geometry">
     <rect>
      <x>1010</x>
      <y>100</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Consolas</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Reset</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget_2">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>30</y>
      <width>311</width>
      <height>191</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777206</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Consolas</family>
     </font>
    </property>
    <property name="dragEnabled">
     <bool>true</bool>
    </property>
    <property name="dragDropOverwriteMode">
     <bool>true</bool>
    </property>
    <property name="dragDropMode">
     <enum>QAbstractItemView::DragDrop</enum>
    </property>
    <property name="defaultDropAction">
     <enum>Qt::MoveAction</enum>
    </property>
    <property name="sortingEnabled">
     <bool>true</bool>
    </property>
    <attribute name="horizontalHeaderVisible">
     <bool>false</bool>
    </attribute>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>50</number>
    </attribute>
    <attribute name="verticalHeaderVisible">
     <bool>false</bool>
    </attribute>
    <row>
     <property name="text">
      <string/>
     </property>
    </row>
    <row>
     <property name="text">
      <string/>
     </property>
    </row>
    <row>
     <property name="text">
      <string/>
     </property>
    </row>
    <row>
     <property name="text">
      <string/>
     </property>
    </row>
    <row>
     <property name="text">
      <string/>
     </property>
    </row>
    <row>
     <property name="text">
      <string/>
     </property>
    </row>
    <column>
     <property name="text">
      <string/>
     </property>
    </column>
    <column>
     <property name="text">
      <string/>
     </property>
    </column>
    <column>
     <property name="text">
      <string/>
     </property>
    </column>
    <column>
     <property name="text">
      <string/>
     </property>
    </column>
    <column>
     <property name="text">
      <string/>
     </property>
    </column>
    <column>
     <property name="text">
      <string/>
     </property>
    </column>
    <item row="0" column="0">
     <property name="text">
      <string/>
     </property>
    </item>
    <item row="0" column="2">
     <property name="text">
      <string/>
     </property>
    </item>
    <item row="0" column="3">
     <property name="text">
      <string/>
     </property>
    </item>
    <item row="0" column="4">
     <property name="text">
      <string/>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="pushButtonKontrola">
    <property name="geometry">
     <rect>
      <x>1010</x>
      <y>170</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Consolas</family>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Kontrola</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>160</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Consolas</family>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Změnit matici</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>130</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Consolas</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Matice 5*5</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>130</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Consolas</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Matice 6*6</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="Execute"/>
  <buttongroup name="Jazyk"/>
 </buttongroups>
</ui>
